examples/query/payment_query.py

```
@@ -1,185 +1,38 @@
-
-def query_payment():
-    """
-    Demonstrates the query payment by:
-    1. Setting up client with operator account
-    2. Creating a fungible token with the operator account as owner
-    3. Demonstrating queries that don't require payment (CryptoGetAccountBalanceQuery)
-    4. Demonstrating queries that do require payment (TokenInfoQuery)
-    5. Comparing network-determined cost vs custom payment amount
-    """
+def query_payment_example():
     client, operator_id, operator_key = setup_client()
```

Ruff (0.14.13)

Unpacked variable `operator_key` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)